Answers.md

```diff
@@ -0,0 +1,9 @@
+What is the difference between an adaptive website and a fully responsive website? Adaptive doesn't have one layout, instead has several layouts for multiple screen sizes. Responsive design aims to provide optimal viewing regardless of the device you view the page on.
+
+Describe what it means to be mobile first vs desktop first. Mobile first means designing the website with a focus on mobile at the start. Desktop would be designing with a desktop focus.
+
+What does font-size: 62.5% in the html tag do for us when using rem units? It will make calculations easier compared to doing say 100%.
+
+How would you describe preprocessing to someone new to CSS? Allows you to store CSS in a variable so you can easily edit just one thing instead of multiple.
+
+What is your favorite concept in preprocessing? What is the concept that gives you the most trouble? My favorite concept would have to be variables & Mixins. I like having to just change one thing instead of every single CSS like before.
```

Comment:
Answers are good enougn, but I would like you to go into a bit more detail in the future 🕵️